### PR TITLE
[MIG] mass_operation_abstract: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**

### CODE REVIEW

File: mass_operation_abstract/wizard/mass_operation_wizard_mixin.py [hunk 1]
```diff
@@ -44,24 +44,26 @@ def default_get(self, fields):
         operation_description_danger = False
 
         # Compute items quantity
-        active_ids = self.env.context.get("active_ids")
+        active_ids = self._context.get("active_ids")
```
Comment (simahawk) [inline mass_operation_abstract/wizard/mass_operation_wizard_mixin.py]: never use a private attr if you have a public one (which is more likely to stay). In any case these changes are not required.

File: mass_operation_abstract/models/mass_operation_mixin.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ class MassOperationMixin(models.AbstractModel):
 
     # To Overwrite Section (Optional)
     def _prepare_action_name(self):
-        return _("Mass Operation (%s)" % (self.name))
+        return _("Mass Operation ({})".format(self.name))
```
Comment (simahawk) [inline mass_operation_abstract/models/mass_operation_mixin.py]: better to keep the old version

**Review by simahawk — CHANGES_REQUESTED** [review]
LG overall

File: mass_operation_abstract/wizard/mass_operation_wizard_mixin.py (hunk 1 above)
Comment (dzungtran89) [reply mass_operation_abstract/wizard/mass_operation_wizard_mixin.py]: thx. I'll update it

**Comment by simahawk** [comment]
sorry I missed to link this PR https://github.com/OCA/server-ux/pull/348 could you have a look and see if we can integrate it here? Otherwise we can open another PR later.

**Comment by dzungtran89** [comment]
hi @simahawk , I ported the update of #348 to this PR, could you pleave have a look?

**Comment by simahawk** [comment]
> hi @simahawk , I ported the update of #348 to this PR, could you pleave have a look?

hmm I don't see the commit :thinking:

**Comment by dzungtran89** [comment]
ah, I actually had appended the update to the migration commit. I just created another commit then

**Comment by simahawk** [comment]
@dzungtran89 you are losing autorship on that commit. Can't you simply cherry-pick? Or use format patch from scratch?

**Comment by dzungtran89** [comment]
Good day @simahawk, thx for the advice, the PR has been updated

**Review by simahawk — APPROVED** [review]

**Review by mmequignon — APPROVED** [review]

**Pull request merged**